hwilib/gui.py

```diff
@@ -207,6 +211,41 @@ def toggle_account(self, checked):
             self.ui.path_lineedit.setEnabled(True)
             self.ui.account_spinbox.setEnabled(False)
 
+class BitBox02PairingDialog(QDialog):
```

Comment:
When is this dialog shown? I can't seem to find a way to show it.

Reply:
Ah, I was able to get it to show if the device was not initialized. However it was just blank until *after* I accepted on the device.

Reply:
The dialog is shown on first pairing, after which the pairing info is persisted and re-used. You can force a new pairing by deleting the pairing info, which on Linux is at `~/.config/bitbox/bitbox02/bitbox02.json` and on macOS is at `~/Library/Application Support/bitbox/bitbox02/bitbox02.json`.

About the blank dialog: it works for me most of the times, but after trying a few more times I can see the issue appear. I'll try to figure out how to make Qt draw the window before proceeding with the pairing.

Reply:
I attempted a fix in `hwilib/gui: make sure the bb02 pairing window is painted hwilib/gui: make sure the bb02 pairing window is painted`. Please try again and lmk if it worked.

Reply:
@achow101 is this resolved for you?

Reply:
Yes, seems to work now.